LICENSE-BUNDLED-DEPENDENCIES

```diff
@@ -2,7 +2,7 @@
 "@babel/code-frame@7.8.3","MIT","https://github.com/babel/babel/tree/master/packages/babel-code-frame"
```

Comment:
Also, I thought we determined that we didn't need this file if we weren't bundling these files with the package.

Reply:
Indeed, I intended to remove it as part of a separate PR